perf/src/index.tsx

```diff
@@ -61,9 +62,15 @@ const renderCycle = async (
   return profilerMeasure
 }
 
-const satisfiesFilter = (componentName: string, filter: string) => {
-  return componentName.toLowerCase().startsWith(filter.toLowerCase())
-}
+const satisfiesFilter = (componentName: string, filter: string) =>
+  minimatch(componentName, filter || '*', {
+    matchBase: true,
```

Comment:
nit: lets be a bit more strict here: we may rename `componentName` with either `componentExampleName` or most explicit `componentFilePath` to better indicate the fact that this is more than just name of the component. It will make it easier to get the reason for this option from code

Reply:
`componentFilePath` is correct 👍